examples/faasd.yaml

```diff
@@ -0,0 +1,48 @@
+# Deploy faasd (which installs a bundled containerd).
+#
+# It can be accessed from the host by authenticating with faas-cli;
+# the ports are already forwarded automatically by lima:
+#
+# $ export OPENFAAS_URL=http://localhost:8080
+# $ limactl shell faasd sudo cat /var/lib/faasd/secrets/basic-auth-password | faas-cli login -u admin --password-stdin
```

Comment:
nit: Could you add how to install `faas-cli`  on the host?

Reply:
done